packages/core/forms/src/components/fields/FieldRadio.vue

```diff
@@ -1,72 +1,97 @@
 <template>
-  <div class="selection-group">
-    <div
+  <div class="radio-selection-group">
+    <KRadio
       v-for="(option, i) in schema.values"
-      :key="i"
-      class="option-group"
-    >
-      <div class="form-group">
-        <label
-          class="k-label"
-          :class="option.name"
-        >
-          <input
-            :id="schema.name+'-'+i"
-            :checked="checkOption(option)"
-            class="k-input"
-            :name="schema.name"
-            type="radio"
-            :value="option.value"
-            @change="onChange"
-          >
-          {{ option.name }}
-        </label>
-      </div>
-    </div>
+      :id="schema.name+'-'+i"
+      :key="option.value"
+      v-model="inputValue"
```

Comment:
Although the `KRadio` worked as expected when we passed `null` to `model-value`, I saw some warnings thrown from it, does it officially support setting the `undefined`/`null` as `model-value`?

```
[Vue warn]: Invalid prop: type check failed for prop "modelValue". Expected String | Number | Boolean | Object, got Null  
  at <KRadio id="gender-0" key="male" modelValue=null  ... > 
  at <FieldRadio ref="child" disabled=null form-options=  ...
  ...
```

Reply:
Addressing in https://github.com/Kong/kongponents/pull/2342
I've also added a test in FieldRadio to verify unsetting works 👍🏻

Reply:
Change has been picked up in the repo